src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/components/step_index_pattern/step_index_pattern.tsx

```diff
@@ -66,6 +63,10 @@ interface StepIndexPatternState {
 }
 
 export class StepIndexPattern extends Component<StepIndexPatternProps, StepIndexPatternState> {
+  static contextType = contextType;
+
+  public readonly context!: KibanaReactContextValue<IndexPatternManagmentContext>;
```

Comment:
I think this should use the `declare context` syntax - 
```
If using the new style context, re-declare this in your class to be the React.ContextType of your static contextType. Should be used with type annotation or static contextType.

static contextType = MyContext
// For TS pre-3.7:
context!: React.ContextType<typeof MyContext>
// For TS 3.7 and above:
declare context: React.ContextType<typeof MyContext>
```

Reply:
looks like `KibanaReactContextValue<IndexPatternManagmentContext>` exists in a number of places. It would be good to define it in a single place.

Reply:
I created type for this.